packages/foreman/foreman/foreman.spec

```diff
@@ -759,7 +760,7 @@ mv Gemfile Gemfile.in
 cp config/database.yml.example config/database.yml
 cp config/settings.yaml.example config/settings.yaml
 export BUNDLER_EXT_GROUPS="default assets"
-ln -s %{nodejs_sitelib} node_modules
+ln -s %{?scl_prefix_nodejs:/opt/theforeman/tfm/root}%{nodejs_sitelib} node_modules
```

Comment:
Isn't there a macro for this root path?

Reply:
Happy to hear it... things get ewird for packages built against but not into the SCL. They don't have all the same macro definitions.

Reply:
Isn't this equal to `%{_scl_root}` since this is built into the tfm SCL?

Reply:
Oooo, maybe! I will give it a try

Reply:
did you?